packages/ui/src/theme/css/component/select.scss

```diff
@@ -61,4 +63,15 @@
       --amplify-components-select-large-padding-inline-end
     );
   }
+
+  &[multiple],
+  &[size] {
+    overflow: auto;
+    padding: 0.25rem 1rem;
+
+    option {
+      color: var(--amplify-components-select-option-color);
```

Comment:
Reminder for us to double check this color property to make sure it works in Firefox + windows/chrome since that has been fussy before.